src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,33 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={classNames({ 'is-active': selectedTab === tab })}
+              data-cy="Tab"
+            >
```

Comment:
In my opinion, we should create the "key" inside this tag
```suggestion
            <li
              className={classNames({ 'is-active': selectedTab === tab })}
              key={tab.id}
              data-cy="Tab"
            >
```

Reply:
> In my opinion, we should create the "key" inside this tag

Good point, `key` is missing